request-subgraph-storage/entrypoint.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+export PATH=$PATH:/usr/local/lib/node_modules
+export NODE_ENV=production
+
+create() {
+    echo "Creating subgraph..."
+    echo -e "\tgraph-node: $GRAPH_NODE"
+    echo -e "\tnamespace: $SUBGRAPH_NAMESPACE"
+    npx graph create \
+        --node "$GRAPH_NODE" \
+        "$SUBGRAPH_NAMESPACE" \
+        --access-token ''
+}
+
+deploy() {
+    local ver=
+    ver=$(version)
```

Comment:
Why not declare and assign in a single statement?
```suggestion
    local ver=$(version)
```

Reply:
this code was under the advice of my vscode bash linter "bash debug".